src/scripts/sync_subclassof.py

```diff
@@ -499,3 +500,4 @@ def run_defaults(use_cache=True):  # todo: #remove-temp-defaults
 
 if __name__ == '__main__':
     cli()
+    #run_defaults()  # TODO temp
```

Comment:
I see that Nico is trying to replicate this issue since I have been unable to ([comment](https://github.com/monarch-initiative/mondo-ingest/issues/406#issuecomment-1916227053)).

FYI Nico this `run_defaults()` is my alternative preferred way to debug / do development on this. You probably already know but I would have changed it back before undrafting this.

Reply:
I know - best not commit development related changes so others can just run the code after checking out the branch. just keep change in "staging"?

Reply:
well, others don't typically need to run my draft code. also i sometimes use more than 1 computer so its good for me to push my state.  but it's not a big deal.

also you too can run it using run_defaults() for much easier debugging if you want ut to automatically pause whenever it encounters an exception, if your setup works like that (I know how to do this in pycharm, not in VS code)

Reply:
Sounds like some good topics to discuss as we are a larger team now.